src/main/java/nl/workingtalent/bookrental/controller/UserController.java

```diff
@@ -114,8 +116,13 @@ public String userLogin(@RequestBody LoginDto loginDto) {
 		String token = RandomStringUtils.random(150, true, true);
 		foundUser.setToken(token);
 		userRepo.save(foundUser);
-					
-		return token;
+		Map<String, String> map = new HashMap<String, String>();	
+		map.put("token", token);
+		
+		if (userIsAdmin(token)) map.put("isAdmin", "admin");
```

Comment:
In theorie kan je bij dit soort if statements, waarbij het 1 executie is voor true, en 1 voor else, het ook kort schrijven met een conditional operator expression. 

Dan zou het er ongeveer zo uit zien:
userIsAdmin ? map.put("isAdmin", "isAdmin") : map.put("isAdmin", "");

de sctructuur van zo'n operator is namelijk:
statement ? executie als de statement TRUE is : executie als de statement false is;

Reply:
@TychoZaal  Klopt. Ik vind dat alleen wel iets onleesbaarder..

> In theorie kan je bij dit soort if statements, waarbij het 1 executie is voor true, en 1 voor else, het ook kort schrijven met een conditional operator expression.
> 
> Dan zou het er ongeveer zo uit zien: userIsAdmin ? map.put("isAdmin", "isAdmin") : map.put("isAdmin", "");
> 
> de sctructuur van zo'n operator is namelijk: statement ? executie als de statement TRUE is : executie als de statement false is;